src/styles/index.css

```diff
@@ -0,0 +1,201 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  --card-width: 200px;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+  padding: 0 50px;
+}
+.header__logo {
+  display: flex;
+}
+.header__logo__img {
+  display: block;
+}
+
+.nav {
+  flex-grow: 1;
+  display: flex;
+  justify-content: flex-end;
+}
+.nav__list {
+  display: flex;
+  list-style-type: none;
+}
+.nav__item {
+  display: flex;
+  align-items: center;
+  height: 100%;
+  text-transform: uppercase;
+}
+.nav__link {
```

Comment:
The color of the navigation links should be `#060b35` instead of `black` as per the task requirements.